smart-contracts/scripts/sendBurnTx.js

```diff
@@ -118,12 +118,25 @@ module.exports = async (cb) => {
     contract.setProvider(web3.currentProvider);
     tokenContract.setProvider(web3.currentProvider);
     try {
+      let isAddress = coinDenom.startsWith("0x")
+      if(!isAddress) {
+        let symbol = coinDenom
+        const instance = await contract.deployed()
+        coinDenom = await instance.getBridgeToken.call(symbol);
+        console.log(symbol + ' address: ' + coinDenom.toString())
+      }
       /*******************************************
        *** Contract interaction
        ******************************************/
       // Get current accounts
       const accounts = await web3.eth.getAccounts();
-      console.log("account is ", accounts[9], cosmosRecipient, coinDenom, amount)
+      let account = accounts[0];
+      if (!DEFAULT_PARAMS) {
+        if (!NETWORK_ROPSTEN && !NETWORK_MAINNET) {
+          account = process.argv[7].toString();
+        }
+      }
+      console.log("account is ", account, cosmosRecipient, coinDenom, amount)
```

Comment:
remove debugging statement

Reply:
Sorry, I didn't get it. in my understanding it is not for debugging. it is just normal printing in console. I made it according to original style of these scripts. Could you clarify on that why you would like to remove it?